### PR TITLE
Adds "Settler" loadout option for Wastelanders

### PR DESCRIPTION
**Pull request by Bauser14**
## About The Pull Request

Introduces a new loadout called "Settler" for Wastelanders that focuses on building and farming. To do this, \Fortune13-master\code\modules\jobs\job_types\wasteland.dm was modified to include a new option in the list of Wastelander loadouts.

## Why It's Good For The Game

Wastelanders are still people who often want stability and safety in their lives, so they commonly take to starting their own homefronts and building whatever habitations they can to survive in peace. I think this addition would give people a new perspective for looking at what they can do as a Wastelander other than just stealing to make a living or trying to wrestle their way into the business of other factions. Plus, it gives raiders somebody else to raid!

## Changelog
:cl:
add: Added "Settler" loadout option for Wastelanders
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -665,6 +666,21 @@ Raider
...
 							/obj/item/gun/ballistic/automatic/pistol/m1911/compact=1)
 
+/datum/outfit/loadout/settler
+	name = "Settler"
+	uniform = /obj/item/clothing/under/f13/settler
+	shoes = /obj/item/clothing/shoes/f13/explorer
+    r_hand = /obj/item/hatchet
+    l_hand = /obj/item/gun/ballistic/automatic/pistol/n99
+    belt = /obj/item/storage/belt
+    backpack_contents = list(/obj/item/stack/sheet/metal=50,
```
Comment (Pebblesthe2nd) [inline code/modules/jobs/job_types/wasteland.dm:676]:
```suggestion
    backpack_contents = list(/obj/item/stack/sheet/metal = 50,
```

File: code/modules/jobs/job_types/wasteland.dm [hunk 2]
```diff
@@ -665,6 +666,21 @@ Raider
...
+                            /obj/item/stack/sheet/mineral/wood=50,
```
Comment (Pebblesthe2nd) [inline code/modules/jobs/job_types/wasteland.dm:677]:
```suggestion
                            /obj/item/stack/sheet/mineral/wood = 50,
```

File: code/modules/jobs/job_types/wasteland.dm [hunk 3]
```diff
@@ -665,6 +666,21 @@ Raider
...
+                            /obj/item/pickaxe/mini=1,
+                            /obj/item/toy/crayon/spraycan=1,
+                            /obj/item/cultivator=1,
+                            /obj/item/reagent_containers/glass/bucket=1,
+                            /obj/item/storage/bag/plants/portaseeder=1)
```
Comment (Pebblesthe2nd) [inline code/modules/jobs/job_types/wasteland.dm:682]:
```suggestion
                            /obj/item/storage/bag/plants/portaseeder = 1,)
```

File: code/modules/jobs/job_types/wasteland.dm [hunk 4]
```diff
@@ -665,6 +666,21 @@ Raider
...
+	shoes = /obj/item/clothing/shoes/f13/explorer
+    r_hand = /obj/item/hatchet
+    l_hand = /obj/item/gun/ballistic/automatic/pistol/n99
+    belt = /obj/item/storage/belt
+    backpack_contents = list(/obj/item/stack/sheet/metal=50,
+                            /obj/item/stack/sheet/mineral/wood=50,
+                            /obj/item/pickaxe/mini=1,
+                            /obj/item/toy/crayon/spraycan=1,
+                            /obj/item/cultivator=1,
+                            /obj/item/reagent_containers/glass/bucket=1,
```
Comment (Pebblesthe2nd) [inline code/modules/jobs/job_types/wasteland.dm:681]:
```suggestion
                            /obj/item/reagent_containers/glass/bucket = 1,
```

File: code/modules/jobs/job_types/wasteland.dm [hunk 5]
```diff
@@ -665,6 +666,21 @@ Raider
...
+	uniform = /obj/item/clothing/under/f13/settler
+	shoes = /obj/item/clothing/shoes/f13/explorer
+    r_hand = /obj/item/hatchet
+    l_hand = /obj/item/gun/ballistic/automatic/pistol/n99
+    belt = /obj/item/storage/belt
+    backpack_contents = list(/obj/item/stack/sheet/metal=50,
+                            /obj/item/stack/sheet/mineral/wood=50,
+                            /obj/item/pickaxe/mini=1,
+                            /obj/item/toy/crayon/spraycan=1,
+                            /obj/item/cultivator=1,
```
Comment (Pebblesthe2nd) [inline code/modules/jobs/job_types/wasteland.dm:680]:
```suggestion
                            /obj/item/cultivator = 1,
```

File: code/modules/jobs/job_types/wasteland.dm [hunk 6]
```diff
@@ -665,6 +666,21 @@ Raider
...
+	name = "Settler"
+	uniform = /obj/item/clothing/under/f13/settler
+	shoes = /obj/item/clothing/shoes/f13/explorer
+    r_hand = /obj/item/hatchet
+    l_hand = /obj/item/gun/ballistic/automatic/pistol/n99
+    belt = /obj/item/storage/belt
+    backpack_contents = list(/obj/item/stack/sheet/metal=50,
+                            /obj/item/stack/sheet/mineral/wood=50,
+                            /obj/item/pickaxe/mini=1,
+                            /obj/item/toy/crayon/spraycan=1,
```
Comment (Pebblesthe2nd) [inline code/modules/jobs/job_types/wasteland.dm:679]:
```suggestion
                            /obj/item/toy/crayon/spraycan = 1,
```

File: code/modules/jobs/job_types/wasteland.dm [hunk 7]
```diff
@@ -665,6 +666,21 @@ Raider
...
+/datum/outfit/loadout/settler
+	name = "Settler"
+	uniform = /obj/item/clothing/under/f13/settler
+	shoes = /obj/item/clothing/shoes/f13/explorer
+    r_hand = /obj/item/hatchet
+    l_hand = /obj/item/gun/ballistic/automatic/pistol/n99
+    belt = /obj/item/storage/belt
+    backpack_contents = list(/obj/item/stack/sheet/metal=50,
+                            /obj/item/stack/sheet/mineral/wood=50,
+                            /obj/item/pickaxe/mini=1,
```
Comment (Pebblesthe2nd) [inline code/modules/jobs/job_types/wasteland.dm:678]:
```suggestion
                            /obj/item/pickaxe/min = 1,
```

**Review by Pebblesthe2nd — CHANGES_REQUESTED** [review]
Minor changes requested to keep to code guidelines. 
PR does what it says. See no reason to not merge, after requested adjustments done.

File: code/modules/jobs/job_types/wasteland.dm (hunk 1 above)
Comment (Rohesie) [reply code/modules/jobs/job_types/wasteland.dm:676]: This is using mixed tab and space indentation.

File: code/modules/jobs/job_types/wasteland.dm (hunk 7 above)
Comment (Bauser14) [reply code/modules/jobs/job_types/wasteland.dm:678]: I'm pretty sure the correct object path is
/obj/item/pickaxe/mini

Based on 
code/modules/mining/equipment/mining_tools.dm

File: code/modules/jobs/job_types/wasteland.dm [hunk 8]
```diff
@@ -665,6 +666,21 @@ Raider
...
+	r_hand = /obj/item/hatchet
+	l_hand = /obj/item/gun/ballistic/automatic/pistol/n99
+	belt = /obj/item/storage/belt
+	backpack_contents = list(/obj/item/stack/sheet/metal = 50,
+							/obj/item/stack/sheet/mineral/wood = 50,
+							/obj/item/pickaxe/mini = 1,
+							/obj/item/toy/crayon/spraycan = 1,
+							/obj/item/cultivator = 1,
+							/obj/item/reagent_containers/glass/bucket = 1,
+							/obj/item/storage/bag/plants/portaseeder = 1,)
```
Comment (Rohesie) [inline code/modules/jobs/job_types/wasteland.dm:682]:
```suggestion
	backpack_contents = list(
		/obj/item/stack/sheet/metal = 50,
		/obj/item/stack/sheet/mineral/wood = 50,
		/obj/item/pickaxe/mini = 1,
		/obj/item/toy/crayon/spraycan = 1,
		/obj/item/cultivator = 1,
		/obj/item/reagent_containers/glass/bucket = 1,
		/obj/item/storage/bag/plants/portaseeder = 1,
		)
```

**Pull request merged**